### PR TITLE
msm8916: surnia: initial bringup to lk2nd

### PR DESCRIPTION
**Pull request by wiktorek140**
Tested, it booting, correct detected under Linux. Can boot other kernel properly.

### CODE REVIEW

File: dts/rules.mk [hunk 1]
```diff
@@ -6,6 +6,7 @@ DTBS += \
 	$(LOCAL_DIR)/apq8016-samsung-r07.dtb \
 	$(LOCAL_DIR)/msm8216-samsung-r05.dtb \
 	$(LOCAL_DIR)/msm8916-motorola-harpia.dtb \
+    $(LOCAL_DIR)/msm8916-motorola-surnia.dtb \
```
Comment (Minecrell) [inline dts/rules.mk:9]: indentation is wrong here :)

Comment (wiktorek140) [reply dts/rules.mk]: done ;)

**Comment by Minecrell** [comment]
@wiktorek140 Can you squash these two commits?

**Comment by wiktorek140** [comment]
@Minecrell yup, done

**Pull request merged**